src/components/Card/index.scss

```diff
@@ -102,6 +102,7 @@
   }
 
   .card-content {
+    overflow-y: auto;
```

Comment:
shouldn't this be `overflow-x`?

Reply:
> shouldn't this be `overflow-x`?

sure, but weird, use `overflow-y` also works, I will change it